uberenv.py

```diff
@@ -748,6 +749,15 @@ def setup_paths_and_dirs(self):
                     print("[WARNING: Could not find Spack Environment file (e.g. spack.yaml) under: {0}]".format(self.spack_env_file))
                     self.spack_env_file = None
 
+        # Copy "defaults.yaml" and "versions.yaml" from configs dir, if they exist
+        for _config_file in ("defaults.yaml", "versions.yaml"):
+          _src = pjoin(spack_configs_path, _config_file)
+          _dst = pabs(pjoin(self.spack_env_directory, "..", _config_file))
+          print("[checking for '{0}' yaml file]".format(_src))
+          if os.path.exists(_src):
+            print("[copying '{0}' config file to {1}]".format(_config_file, _dst))
+            sexe("cp {0} {1}".format(_src, _dst))
```

Comment:
```suggestion
            shutil.copy(_src, _dst)
```

Reply:
If we run uberenv a few times, will it always wipe out previous copy of `versions.yaml` and `defaults.yaml`?
I'll merge this and try it out, and revert if it's a problem.

Reply:
oh thats a good point... just change it to `shutil.copy`

Reply:
I confirmed that it threw the error on subsequent runs and removed the check:
```
[copying 'defaults.yaml' config file to .../uberenv_libs/defaults.yaml]
Traceback (most recent call last):
  File "..././scripts/uberenv/uberenv.py", line 1370, in <module>
    sys.exit(main())
  File "..././scripts/uberenv/uberenv.py", line 1305, in main
    env.setup_paths_and_dirs()
  File "..././scripts/uberenv/uberenv.py", line 760, in setup_paths_and_dirs
    raise FileExistsError(f"The file '{_dst}' already exists.")
FileExistsError: The file '.../uberenv_libs/defaults.yaml' already exists.
```